### ISSUE TITLE
Timeline moveTo it's not centering

### ISSUE THREAD

**Issue by Kianda** [almende/vis#1674]
"Move the window such that given time is centered on screen."

http://pastebin.com/NRcF1J0r

Copy my code and try, the screen it's not centered on:
timeline.moveTo('2016-02-20T12:00:00');

I'm doing something wrong?
Thanks!

**Comment by josdejong** [comment]
Thanks for reporting this. It looks like the `moveTo` is applied before the Timeline has set it's initial window (based on data, zoomMin, zoomMax, etc).

The following will work as expected.

```
var timeline = new vis.Timeline(container, items, groups, options);
timeline.moveTo('2016-02-20T12:00:00');
```

I will have a look into whether we can change `setItems` and `setGroups` to be applied synchronously too, that would be nicer and is what we would expect here.

**Comment by Kianda** [comment]
Thanks!!

**Comment by mojoaxel** [comment]
@yotamberk Are you sure this is fixed? Maybe it's a bug?

**Comment by yotamberk** [comment]
@mojoaxel  it worked for me. Unless I did something wrong, I think this issue has been resolved.

**Comment by JoseCunha** [comment]
Hi, 

Don't know if it's the correct place to ask about this issue again, so apologies for that if it's not the case.

I have the following case for VISJS Timeline version 4.20.1:

I created the timeline like this:
var timeline = new vis.Timeline(container);
I do some operations to obtain the options and later on I add them to the Timeline:

var options = {max:new Date(2018, 1, 1),min:new Date(1980,1,1), zoomMax:315360000000000, zoomMin:10, zoomable:true}
timeline.setOptions(options);

I added several items to near the begin (min) and the end (max) of the timeline.

After that I use the method moveTo to move and center to a specific date. 
In the majority of the cases it works as expected, but when it approaches the limits (max or min) it doesn't center and sometimes the callback doesn't work.

In a small example I present to you this case, but it's only as an example.:
[Timeline_moveTo_example.zip](https://github.com/almende/vis/files/1295836/Timeline_moveTo_example.zip)
 
Thank you in advance.
Best regards,
José Cunha
